lab-bundle-spec.md

```diff
@@ -271,13 +269,92 @@ No additional attributes
   id: primary_domain
 ```
 
-##### Future Resource Types
+##### AWS Account (aws_account)
+
+attribute   | required | type       | notes
+----------- | -------- | ---------- | ----------------------------------------
+default_policy.allow_aws_dedicated_instances |          | boolean | Default false.
+default_policy.allow_aws_ondemand_instances  |          | array   | Array of [EC2 instance types](#valid-eC2-instance-types) that are valid Ondemand usage. Default none.
+default_policy.allow_aws_rds_instances       |          | array   | Array of [EC2 instance types](#valid-eC2-instance-types) that are valid for RDS usage. Default none.
+default_policy.allow_aws_spot_instances      |          | boolean | Default false.
+default_policy.allow_aws_subnet_deletion     |          | boolean | Default false.
+default_policy.allow_aws_vpc_deletion        |          | boolean | Default false.
+startup_script.type              |          | string  | The type of startup script. Only `cloud_formation` is supported.
+startup_script.path              |          | path    | Relative path to a directory tree with the script contents.
+startup_script.custom_properties |          | array   | Array of pairs. See [Custom Script Properties](#custom-script-properties) for details.
+cleanup_script.type              |          | string  | The type of cleanup script. Only `cloud_formation` is supported.
+cleanup_script.path              |          | path    | Relative path to a directory tree with the script contents.
+cleanup_script.custom_properties |          | array   | Array of pairs. See [Custom Script Properties](#custom-script-properties) for details.
+user_policy                      |          | path    | Relative path to a [JSON policy](https://awspolicygen.s3.amazonaws.com/policygen.html) document
+variant                          |          | enum*   | Specify a Qwiklabs fleet name.
+
+```yml
+- type: aws_account
+  id: the_account
+  variant: sts
+  startup_script:
+    type: cloud_formation
+    path: ./lab.template
+    custom_properties:
+      - key: userNameWindows
+        value: student
+  user_policy: ./student.policy
+  default_policy:
```

Comment:
why are these under `default_policy`? i know that's a term from AWS infra, but not sure how useful it is here. also, should/could we have a single `policies` header that accounts for the below and the `user_policy`?

Reply:
Good question. I'll add some more documentation about what this section actually does. But to answer you directly, there are few important clarifying points:

`default_policy` can be thought of as the account level policies. 
The `default_policy` input is not structured as a normal policy file. These values are flags that Qwiklabs uses to construct policy.
Lab authors are not allowed to directly specify this policy, because there are a number of properties that Qwiklabs needs to ensure are always set (e.g. settings that prevent abusers from taking full control of the account and breaking the cleaning cycle).

Reply:
Maybe rename it to `account_restrictions` or something, so people don't get confused with the full `default_policy` concept? Or `default_policy_flags` or `default_policy_args`?

Reply:
Oooh... I like both `account_restrictions` and `default_policy_flags`. I'll go with `account_restrictions` because it is a bit more flexible in usage; but I'll happily exchange it for the alternative.